app/views/interviews/index.html.erb

```diff
@@ -0,0 +1,51 @@
+<div class="container">
+  <div class="row">
+
+      <% @interviews.each do |interview| %>
+        <div class="card col-12 col-sm-6 col-md-4 col-lg-3" style="width: 18rem; height: 20rem">
+          <div class="d-flex">
+            <div>
+              <!-- Icon Clock -->
+            </div>
+            <div>
+              <%= interview.time.strftime("%a %e.%m.%y | %l:%M") %>
+            </div>
+          </div>
+
+          <div>
+            <div>
+              <!-- Icon Code -->
+            </div>
+            <div>
+              <!-- Tags -->
```

Comment:
For today's demo, should we just add the tags as a list here?

Reply:
OK.